kover-jvm-agent/docs/index.md

```diff
@@ -0,0 +1,45 @@
+# Kover JVM instrumentation agent
+
+## On the fly instrumentation
+On the fly instrumentation - modification of the bytecode of classes in order to measure coverage, that occurs when the class is loaded into the JVM.
+
+To instrument the loaded classes, it is necessary to connect a JVM agent to the Java application being launched.
+
+## Getting a readable report
+To get a readable coverage report, you need to:
+1. [Connect JVM agent on JVM start](#connecting-the-jvm-agent)
+2. Terminate the JVM application. When application finished, the binary report file will be saved
+3. Using the [Kover CLI](/cli#generating-reports), generate a human-readable report from a binary report
```

Comment:
Is `cli/#...` link correct? It is not resolvable in IntelliJ

Reply:
It should be correct after moving the documentation to a single location